runner/ansible/roles/2.1.6/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
Hey @brett060102 ,
I have tested this `2.1.6`, and I would say it is broken, as the queried lines return multi line outputs, so the comparisons always fail
Here my outputs using the command that we have right now:
```
vmhana01:/home/cloudadmin # cibadmin -Q --xpath "//constraints/rsc_colocation" | grep -oP 'score="\K[^"]+'
4000
-INFINITY
vmhana01:/home/cloudadmin # cibadmin -Q --xpath "//constraints/rsc_colocation" | grep -oP 'rsc="\K[^"]+' | grep -o g_ip
g_ip
vmhana01:/home/cloudadmin # cibadmin -Q --xpath "//constraints/rsc_colocation" | grep -oP 'rsc-role="\K[^"]+'
Started
Master
Started
Slave
vmhana01:/home/cloudadmin # cibadmin -Q --xpath "//constraints/rsc_colocation" | grep -oP 'with-rsc="\K[^"]+' | grep -o msl_SAPHana
msl_SAPHana
msl_SAPHana
vmhana01:/home/cloudadmin # cibadmin -Q --xpath "//constraints/rsc_colocation" | grep -oP 'with-rsc-role="\K[^"]+'
Master
Slave
vmhana01:/home/cloudadmin # cibadmin -Q --xpath "//constraints/rsc_order" | grep -oP 'kind="\K[^"]+'
Optional
vmhana01:/home/cloudadmin # cibadmin -Q --xpath "//constraints/rsc_order" | grep -oP 'first="\K[^"]+' | grep -o cln_SAPHanaTopology
cln_SAPHanaTopology
vmhana01:/home/cloudadmin # cibadmin -Q --xpath "//constraints/rsc_order" | grep -oP 'then="\K[^"]+' | grep -o msl_SAPHana
msl_SAPHana
vmhana01:/home/cloudadmin # 

```

Reply:
Thanks, I see that you added a space to the initial grep search string. Not sure how this worked in the example code though.